core/ExecutionSummary.go

```diff
@@ -0,0 +1,29 @@
+package core
+
+type MinMaxMeanTotalInt struct {
```

Comment:
I would call this type Stat or something.  If we add StdDev we then have to change the name of the type.